weetweet.py

```diff
@@ -288,19 +291,35 @@ def trim_tweet_data(tweet_data, screen_name, alt_rt_style, is_py3):
                 if message['user']['screen_name'] == screen_name:
                     #escape highlighting
                     message['user']['screen_name'] = "<you>"
-                message['text'] = message['retweeted_status']['text'] + " (retweeted by " + message['user']['screen_name'] + ")"
+                if message.get('full_text'):
+                    message['full_text'] = message[
+                        'retweeted_status']['full_text'] + " (retweeted by " + message[
+                            'user']['screen_name'] + ")"
+                else:
+                    message['text'] = message['retweeted_status']['text'] + \
+                            " (retweeted by " + message['user']['screen_name'] + ")"
                 message['user'] = message['retweeted_status']['user']
             else:
-                message['text'] = "RT @{}: {}".format(message['retweeted_status']['user']['screen_name'],
-                                                      message['retweeted_status']['text'])
-        mes_list = [calendar.timegm(time.strptime(message['created_at'],'%a %b %d %H:%M:%S +0000 %Y')),
-            message['user']['screen_name'],
-            message['id_str']]
+                if message.get('full_text'):
+                    message['full_text'] = "RT @{}: {}".format(message['retweeted_status']['user']\
+                            ['screen_name'], message['retweeted_status']['full_text'])
+                else:
+                    message['text'] = "RT @{}: {}".format(message['retweeted_status']['user']\
+                            ['screen_name'], message['retweeted_status']['text'])
+        mes_list = [calendar.timegm(time.strptime(
+            message['created_at'], '%a %b %d %H:%M:%S +0000 %Y')),
+                    message['user']['screen_name'], message['id_str']]
```

Comment:
I think this only makes it harder to see where the arguments would end up in `mes_list`. Before you could at a quick glance see that it would (probably) only be three.